pluton/cluster.go

```diff
@@ -43,6 +47,7 @@ type Info struct {
 	KubeletTag      string // e.g. v1.5.3_coreos.0
 	Version         string // e.g. v1.5.3+coreos.0
 	UpstreamVersion string // e.g. v1.5.3
+	Options                // pipe through how test was configured
```

Comment:
I don't see any sign of this actually being used. So you could just drop this and move the Options attributes to the Test struct and simplify declarations.